package.json

```diff
@@ -30,20 +30,21 @@
   },
   "homepage": "https://github.com/rollup/rollup-plugin-svelte#README",
   "devDependencies": {
-    "eslint": "^4.12.1",
-    "locate-character": "^2.0.3",
-    "mocha": "^4.0.1",
-    "rollup": "^0.52.0",
-    "rollup-plugin-buble": "^0.18.0",
+    "eslint": "^5.0.1",
+    "locate-character": "^2.0.5",
+    "mocha": "^5.2.0",
+    "rollup": "^0.62.0",
+    "rollup-plugin-buble": "^0.19.2",
     "rollup-watch": "^4.3.1",
     "sander": "^0.6.0",
-    "source-map": "^0.6.1",
-    "svelte": "^1.60.0"
+    "source-map": "^0.7.3",
```

Comment:
looks like this needs to be locked to 0.6, as they changed the API (alternatively we could change our usage of the library) — the tests are failing

Reply:
I've reverted the version update. Will treat updating this as a separate PR.. don't want this to creep into a whole rewrite :)